src/styles/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 5px;
+  padding-left: 16px;
+  padding-right: 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-image: url("../images/imac.jpeg");
+    background-size: 160px 134px;
+    margin-top: 32px;
+  }
+
+  &__text {
+    display: block;
+  }
+
+  &__text--name {
+    color: $main-accent;
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__text--product-code {
+    margin-top: 4px;
+    color: #616070;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    @include content-display;
+
+    width: 100%;
+    align-items: flex-end;
+    line-height: 14px;
+    margin-top: 16px;
+  }
+
+  &__reviews-amount {
+    color: $main-accent;
+    font-size: 10px;
+  }
```

No need to duplicate selectors
```suggestion
  &__reviews {
    @include content-display;

    width: 100%;
    align-items: flex-end;
    line-height: 14px;
    margin-top: 16px;
    
    &-amount {
      color: $main-accent;
      font-size: 10px;
    }
  }
```